va/Android.mk

```diff
@@ -25,8 +25,8 @@
 
 LOCAL_PATH:= $(call my-dir)
 
-LIBVA_DRIVERS_PATH_32 := /vendor/lib/dri
-LIBVA_DRIVERS_PATH_64 := /vendor/lib64/dri
+LIBVA_DRIVERS_PATH_32 := /vendor/lib/:/system/lib
```

Comment:
I wonder whether it will impact old setting

Reply:
none impact, because it will use /system/lib or /system/lib64 to find share library，I can add dri, but it can't used on android.